test/cluster/test_view_building_coordinator.py

```
@@ -40,41 +40,41 @@
     servers = await manager.running_servers()
     await asyncio.gather(*(manager.api.enable_injection(s.ip_addr, VIEW_BUILDING_COORDINATOR_PAUSE_MAIN_LOOP, one_shot=True) for s in servers))
 
-async def unpause_view_build_coordinator(manager: ManagerClient):
+async def unpause_view_build_coordinator(manager: ScyllaClusterManager):
     """Unpause the view build coordinator."""
     servers = await manager.running_servers()
     await asyncio.gather(*(manager.api.message_injection(s.ip_addr, VIEW_BUILDING_COORDINATOR_PAUSE_MAIN_LOOP) for s in servers))
     await asyncio.gather(*(manager.api.disable_injection(s.ip_addr, VIEW_BUILDING_COORDINATOR_PAUSE_MAIN_LOOP) for s in servers))
 
-async def pause_view_building_tasks(manager: ManagerClient, token: int | None = None, pause_all: bool = True):
+async def pause_view_building_tasks(manager: ScyllaClusterManager, token: int | None = None, pause_all: bool = True):
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)